azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/VideoViewManager.kt

```diff
@@ -114,14 +124,15 @@ internal class VideoViewManager(
                 if (stream != null) {
```

Comment:
In kotlin, it's probably better to use .apply {} or .let {} as opposed to null check, because you can use "this" or "it" without the !! to access the object in a null safe way.

e.g. 
```stream?.let {
 //it = stream, and not null
}```

Reply:
I guess it's not really part of this PR though at all. Just noticed the !! in the file.